src/test/java/CreatePlayList.java

```diff
@@ -0,0 +1,136 @@
+        //REPEAT
+        playList.playListArea();
+        playList.addingPlaylistbtn();
+        enterPlayListName("second");
+        //EXPECTED TO BE FALSE
+        Assert.assertEquals(getConfirmationMessage(),false,"CANNOT ENTER SAME NAME");
+    }
+
+    @Test(enabled = true, priority = 3)
+    public void createNameWith10Chars () throws InterruptedException {
+        //GIVEN
+        AutoLoginPage loginPage = new AutoLoginPage(driver);
+        AutoHomePage homePage = new AutoHomePage(driver);
+        AutoPlaylistPage playList = new AutoPlaylistPage(driver);
+        loginPage.login();
+        Assert.assertTrue(homePage.isUserAvatarDisplayed());
+        //WHEN
+        playList.playListArea();
+        playList.addingPlaylistbtn();
+        enterPlayListName(tenChars);
+        //THEN
+        Assert.assertEquals(getConfirmationPopupText(), "Created playlist " + "\"" + tenChars + "." + "\"");
+    }
+    @Test(enabled = true, priority = 4)
+    public void createNameWith11Chars () throws InterruptedException {
+        //GIVEN
+        AutoLoginPage loginPage = new AutoLoginPage(driver);
+        AutoHomePage homePage = new AutoHomePage(driver);
+        AutoPlaylistPage playList = new AutoPlaylistPage(driver);
+        loginPage.login();
+        Assert.assertTrue(homePage.isUserAvatarDisplayed());
+        //WHEN
+        playList.playListArea();
+        playList.addingPlaylistbtn();
+        enterPlayListName(elevenChars);
+        //THEN
+        Assert.assertEquals(getConfirmationMessage(),false);
+    }
+    @Test(enabled = true, priority = 3)
+    public void createNameTwoChars () throws InterruptedException {
+        //GIVEN
+        AutoLoginPage loginPage = new AutoLoginPage(driver);
+        AutoHomePage homePage = new AutoHomePage(driver);
+        AutoPlaylistPage playList = new AutoPlaylistPage(driver);
+        loginPage.login();
+        Assert.assertTrue(homePage.isUserAvatarDisplayed());
+        //WHEN
+        //THEN
+        playList.playListArea();
+        playList.addingPlaylistbtn();
+        enterPlayListName(two);
+        Assert.assertEquals(getConfirmationMessage(),false);
+    }
+
+    public void enterPlayListName(String playListName) {
+        WebElement enterPlaylistName = driver.findElement(By.cssSelector("input[name='name']"));
+        enterPlaylistName.sendKeys((Keys.chord(Keys.COMMAND, "a", Keys.BACK_SPACE)));
+        enterPlaylistName.sendKeys(playListName);
+        enterPlaylistName.sendKeys(Keys.ENTER);
+    }
+    private String getConfirmationPopupText() {
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("div.success.show")));
+        return driver.findElement(By.cssSelector("div.success.show")).getText();
+    }
+    private boolean getConfirmationMessage() {
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("div.success.show")));
+        return driver.findElement(By.cssSelector("div.success.show")).isDisplayed();
+    }
```

Comment:
![image](https://user-images.githubusercontent.com/104761482/211289353-1f7e2582-26f7-4e21-b7e2-6ce441b1bc0b.png)

Reply:
I moved all helping methods to Auto.Pages.